rpa-builder/6.6/modules/ROOT/pages/toolbox-file-operations-extract-pages-from-pdf-file.adoc

```diff
@@ -0,0 +1,62 @@
+= Extract Pages from PDF File
+
+Extract pages from a PDF file, customize the output, and create a new PDF file with the configured settings.
+
+== Properties
+
+* *File Settings*
+* *Create Target PDF if it doesn't exist*
++ 
+Select this option to create the PDF file if the specified output file name doesn't exist. Otherwise, the action step fails if the file doesn't exist. 
+* *Extraction Page Range* 
++
+The pages to extract from the input PDF. Use commas (`,`) to separate multiple ranges and `EOF` to specific the end of file. For example: `3-4, 8, 11-EOF`.

```

Comment:
I think it's worth mentioning here that you can also put in reverse order ranges. For example, a range of 5-3 will copy the pages 5, 4, 3 in that order into the new document. This also works with EOF, so if you wanted to reverse an entire PDF file, you can put in EOF-1.

Reply:
Fixed